frontend/app/assets/javascripts/top_containers.bulk.js

```diff
@@ -103,8 +103,50 @@ BulkContainerSearch.prototype.update_button_state = function() {
 };
 
 BulkContainerSearch.prototype.setup_table_sorter = function() {
-  function padValue(value) {
-    return (new Array(255).join("#") + value).slice(-255)
+  function padNumber(number) {
+    // Get rid of preceding zeros from numbers (so 003 will sort with 3 instead of in the hundreds)
+    // Then pad it (so 10 doesn't sort between 1 and 2)
+    number = parseInt(number).toString()
+    return  (new Array(255).join("#") + number).slice(-255)
+  }
+
+  function parseIndicator(value) {
+    // Creates a string of alternating number/non-number values separated by commas for indicator sort
+    if (!value || value.length === 0) {
+      return value
+    }
+
+    let isNumber = isNaN(parseInt(value[0])) ? false : true
+
+    let valueArray = [value[0]]
+    let valueArrayCurrentIndex = 0;
+    for (i = 1; i < value.length; i++) {
+      if (!isNumber) {
+        if (isNaN(parseInt(value[i]))) {
+          valueArray[valueArrayCurrentIndex] += value[i]
+        } else {
+          valueArray[valueArrayCurrentIndex] = valueArray[valueArrayCurrentIndex].trim()
+          valueArrayCurrentIndex += 1
+          valueArray[valueArrayCurrentIndex] = value[i]
+          isNumber = true
+        }
+      } else {
+        if (isNaN(parseInt(value[i]))) {
+          valueArray[valueArrayCurrentIndex] = padNumber(valueArray[valueArrayCurrentIndex])
+          valueArrayCurrentIndex += 1
+          valueArray[valueArrayCurrentIndex] = value[i]
+          isNumber = false
+        } else {
+          valueArray[valueArrayCurrentIndex] += value[i]
+        }
+      }
+    }
+
+    if (!isNaN(parseInt(valueArray[valueArray.length - 1]))) {
+      valueArray[valueArray.length - 1] = padNumber(valueArray[valueArray.length - 1])
+    }
+
+    return valueArray.toString()
```

Comment:
This logic seems to have problems with numerical values.  I tested some values using JSFiddle and did not get the expected output.  Here are the results of my tests: https://jsfiddle.net/ives1227/fo4ywbqk/

I'm happy to help look into this more.